src/main/java/com/zaxxer/hikari/pool/PoolElf.java

```diff
@@ -281,15 +284,6 @@ void resetConnectionState(final PoolBagEntry poolEntry) throws SQLException
       }
    }
 
-   void resetPoolEntry(final PoolBagEntry poolEntry)
-   {
-      poolEntry.setReadOnly(isReadOnly);
-      poolEntry.setCatalog(catalog);
-      poolEntry.setAutoCommit(isAutoCommit);
-      poolEntry.setNetworkTimeout(networkTimeout);
-      poolEntry.setTransactionIsolation(transactionIsolation);
-   }
-
```

Comment:
BTW renaming method is not going to solve mess of back and forth method calls.
Thanks